code/game/jobs/job/job.dm

```diff
@@ -30,10 +30,22 @@
 
 /datum/job/proc/equip_backpack(var/mob/living/carbon/human/H)
 	switch(H.backbag)
-		if(2) H.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack(H), slot_back)
-		if(3) H.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack/satchel_norm(H), slot_back)
-		if(4) H.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack/satchel(H), slot_back)
-		if(5) H.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack/duffel(H), slot_back)
+		if(2)
+			var/obj/item/weapon/storage/backpack/B = new /obj/item/weapon/storage/backpack(H)
+			H.equip_to_slot_or_del(B, slot_back)
+			B.autodrobe_no_remove = 1

```

Comment:
These can be deleted as well. Basically, you're calling equip\_to\_slot\_or\_**del**. Gotta prepare for when it actually deletes shit.

```
if (!QDELETED(B))
    B.autodrobe_no_remove = 1
```